browser/src/core/LOUtil.js

```diff
@@ -118,6 +118,28 @@ L.LOUtil = {
 
 		map.on('themechanged', setupIcon, this);
 	},
+	setUserImage: function(img, map) {
+		var docType = map.getDocType();
+		// set avatar image if it exist in user extract info
+		var defaultImage = L.LOUtil.getImageURL('user.svg', docType);
+		var myViewId = map._docLayer._viewId;
+		var extraInfo = map._viewInfo[myViewId].userextrainfo;
+		if (extraInfo !== undefined && extraInfo.avatar !== undefined) {
+			// set user avatar
+			img.src = extraInfo.avatar;
+			// Track if error event is already bound to this image
+			if (!img.errorEventBound) {
+				img.errorEventBound = true;
+				img.addEventListener('error', function() {
+					img.src = defaultImage;
+					this.checkIfImageExists(img, true);
+				}.bind(this));
+			}
+			return;
```

Comment:
setUserImage is unlikely to be called twice, so this doesn't matter... I presume this is to avoid problems with the error event executing multiple times in which case you likely want something that modifies the event itself like so...
```suggestion
			// Track if error event is already bound to this image
			img.addEventListener('error', () => {
				if (e.loUtilProcessed) {
					return;
				}

				img.src = defaultImage;

				e.loUtilProcessed = true;
				// FIXME: remove this error event listener here to avoid future conflicts with the checkIfImageExists listener

				this.checkIfImageExists(img, true);
			});
			return;
```

That also has the advantage of hooking into the same thing that the checkIfImageExists error event does, so the same event retriggering also won't cause an issue there.

You would also need to do the thing in the FIXME comment to avoid future errors rerunning this error event listener.